### ISSUE TITLE
Make.md breaks plugin

### ISSUE THREAD

**Issue by ashworksco** [kelszo/obsidian-file-explorer-plus#5]
### What happened?

Files do not get pinned or hidden, but the plugin thinks they have.

### What did you expect to happen?

I expect files to either be pinned or hidden based on right clicking and selecting from the pop-up menu.

### Can you reproduce the bug, if so how? As detailed but minimal as possible.

Yes, and I have done so several times now with installing/uninstalling.

### Plugin version

1.1.3

### Obsidian version

1.5.3

### OS

macOS

**Comment by kelszo** [comment]
Can you provide some more details? Currently working as expected on both my devices, cannot replicate

**Comment by kelszo** [comment]
According to the latest Changelog for Obsidian:
> File Explorer and Tags pane both received a small code refactor. This might affect some plugins that rely on the internal JS structure. Themes should not be affected.

However, I am currently also running Obsidian v1.5.3 and it works. Do you have any other plugins that affect the File Explorer? Has the plugin worked before?

**Comment by kelszo** [comment]
Thank you for the vital information @zenus260!  

I tested installing make.md and I found that File Explorer++ stopped working on reload. If @ashworksco can confirm they are also running make.md I think we can confirm the cause.

Make.md seems to be a really cool plugin, wouldn't mind testing myself. However, looking through their code they have some aggressive code practice. Seems like the cause is that they override the base Obsidian function `getLeavesOfType` so when trying to fetch the File Explorer leaf it returns their leaf instead. See: [patches.ts](https://github.com/Make-md/makemd/blob/1560cf1d522eebabd5a8b36a943f296ece1069df/src/adapters/obsidian/utils/patches.ts#L18) in the make.md code. 

TLDR: To be honest, nothing we can do on our side. They make it impossible to fetch the file-explorer leaf and thus break our integration. The make.md team should consider a less invasive approach to their patching method for interoperability with other plugins.
